API Users, reorganize, implement fontawesome to remove SVG images

vsc55 opened the pull request:
Add support to github url with the user's profile and it will show the information.
Rearrange files.
Delete SVG files, now the images are class fontawesome.

vsc55 committed:
Merge pull request #1 from audrasjb/master

Update

vsc55 committed:
correct spelling error.

add text that did not implement i18n/l10n.

vsc55 committed:
Add image in links.

Sort style sheets and delete code not necessary.

vsc55 committed:
Add box when an error message is generated.

Detect if the url is not from a repository and corresponds to a user, in this case it shows the user's information.
Create an external object to manage transitions.

vsc55 committed:
Update info-user template.

Rename template repository to info-repo.
Control darkmode in info-user edition mode.
Have the collection of data from the github api that will be used for the replacements in the templates be dynamically added.

vsc55 committed:
organize code

name change "dark-mode" by "dark-theme"fork
change image on_off

vsc55 committed:
Delete SVG files, now the images are class fontawesome.

Finish adjusting users template.
Update all templates.

vsc55 committed:
fix

vsc55 committed:
Hide "Starter" in the user information, it does not work yet.

vsc55 commented:
Fix issues #5

audrasjb commented:
Hi,
Thank you for this pull request @vsc55 !
The support for user's profile looks great, but I'd prefer to keep SVG images instead of using icons from an external library (font awesome). Could you please propose the first feature separately?

Thanks!
Jb

vsc55 commented:
Hi,
OK no problem.
And how about having both options?
Just add a check to select if you want to use SVG images or fontawesome icons.


Best regards.

vsc55 committed:
API Users

Update Master API Users

vsc55 committed:
files svg recovery

Add option for select the source icon file svg or Font Awesome.
Pending implement select source icon.

vsc55 committed:
Add images for user template.

Add option in config block for select images source.

vsc55 committed:
Add suport files SVG and Font Awesome

Add images for user template.
Add option in config block for select images source.

vsc55 commented:
It's ready.
Although the images of Followers, Following, Public Repo and Pubic Gist do not know if they like them very much.

vsc55 committed:
wipe code

vsc55 committed:
wipe code

vsc55 committed:
Fix missing css github icon.

vsc55 committed:
Merge pull request #5 from vsc55/master

Fix missing css github icon.

codercampos commented:
If this solves issue #5 I can close PR #6 after merged.
